src/qthotkey/plugin.h

```diff
@@ -0,0 +1,50 @@
+#ifndef _PLUGIN_H_INCLUDED_
+#define _PLUGIN_H_INCLUDED_
+
+#include <QtCore/QList>
+
+typedef enum {
```

Comment:
typedef syntax here is redundant, use `enum Event { ... };`.

Reply:
Thanks, fixed it here.